### DISCUSSION TITLE
If we enable "fast-diff" features, is there bad effects for images?

### DISCUSSION THREAD

**Discussion by zhucan** [ceph/ceph-csi#3433] [Q&A]
Because of the cmd "rbd du" of the image that has no "fast-diff" enable.

**Comment by Madhu-1** [comment]
These image features are enabled by default even in RBD and AFAIK doesnt have any bad effects. some details about image features are available here https://tracker.ceph.com/issues/15000

**Reply by zhucan** [reply]
if there is  no bad effects, why ceph csi not enable it default?

**Reply by Madhu-1** [reply]
cannot be enabled by default because of the kernel client dependency old kernel like (3.x) cannot map the rbd images with this feature you need 5.4+ kernel  that why its options, if the user wants he can enable it my default. see https://docs.ceph.com/en/latest/rbd/rbd-config-ref/#image-features for more details.

**Reply by zhucan** [reply]
Thanks, copy that @Madhu-1
